Fix XREFs (#2084)
theforeman/foreman-documentation: guides/common/modules/proc_enabling-the-satellite-tools-repository.adoc

== [[enabling-repos-rhel9-rhel8]]{RHEL} 9 & {RHEL} 8

> To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Enabling_the_Client_Repository_rhel_9_{context}[Enabling the Client Repository on RHEL 9] or 
> xref:CLI_Enabling_the_Client_Repository_rhel_8_{context}[Enabling the Client Repository on RHEL 8].

Reviewer: ```suggestion
To use the CLI instead of the {ProjectWebUI}, see the procedure relevant for your {RHEL} version:
* xref:CLI_Enabling_the_Client_Repository_rhel_9_{context}[]
* xref:CLI_Enabling_the_Client_Repository_rhel_8_{context}[]
```

For more information, see https://www.redhat.com/en/resources/els-datasheet[{RHEL} Extended Life Cycle Support (ELS) Add-on] guide.
====

> To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Enabling_the_Client_Repository_rhel_7_{context}[Enabling the Client Repository on RHEL 7] or 
> xref:CLI_Enabling_the_Client_Repository_rhel_6_{context}[Enabling the Client Repository on RHEL 6].

Reviewer: ```suggestion
To use the CLI instead of the {ProjectWebUI}, see the procedure relevant for your {RHEL} version:
* xref:CLI_Enabling_the_Client_Repository_rhel_7_{context}[]
* xref:CLI_Enabling_the_Client_Repository_rhel_6_{context}[]
```
